src/main/java/ca/corefacility/bioinformatics/irida/ria/web/users/dto/PasswordResetResponse.java

```diff
@@ -0,0 +1,32 @@
+package ca.corefacility.bioinformatics.irida.ria.web.users.dto;
+
+import java.util.Map;
+
+/**
+ * Data transfer object for password reset response
```

Comment:
These types of responses are hard to use in the UI.  It is better to return a success code with an `AjaxSuccessResponse` or and `AjaxFormErrorResponse` (handles the map of different messages) with and http error response.

Reply:
Updated in 66fd23c47ca7b634ae1918ac77341a2fdf53048e